src/backend/eventEmitters/consoleEmitter.ts

```diff
@@ -1,11 +0,0 @@
-/* eslint-disable class-methods-use-this */
-import { BudgetTrackingEventEmitter, EventPublisher } from '@/backend/eventEmitters/EventEmitter';
-
-export function buildConsoleEmitter(): EventPublisher {
-  const consoleEmitter = new BudgetTrackingEventEmitter();
-  consoleEmitter.onAny((eventName, eventData) => {
-    // eslint-disable-next-line no-console
-    console.log(`${eventName}:`, eventData);
-  });
-  return consoleEmitter;
-}
```

Comment:
As backend (think about the option to implement CLI in the future, for example), you not sure you will print to the console, you leave it to the "backend user" decision.

Reply:
But see the new `loggerEmitter`, it is very similar.